quickstart/201-iot-hub-with-device-provisioning-service/main.tf

```diff
@@ -0,0 +1,165 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  location = var.resource_group_location
+  name     = random_pet.rg_name.id
+}
+
+# Create storage account & container
+resource "random_string" "sa_name" {
+  length = 12
+  special = false
+  upper = false
+}
+
+resource "azurerm_storage_account" "sa" {
+  name                     = random_string.sa_name.id
+  resource_group_name      = azurerm_resource_group.rg.name
+  location                 = azurerm_resource_group.rg.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_storage_container" "my_terraform_container" {
+  name                  = "mycontainer"
+  storage_account_name  = azurerm_storage_account.sa.name
+  container_access_type = "private"
+}
+
+
+# Create an Event Hub & Authorization Rule
+resource "random_pet" "eventhubnamespace_name" {
+  prefix = var.eventhub_namespace_name_prefix
+}
+
+resource "azurerm_eventhub_namespace" "namespace" {
+  name                = random_pet.eventhubnamespace_name.id
```

Comment:
Would `random_pet`'s name `eventhub_namespace_name` be better here?

Reply:
@lonegunmanb I'm not sure what this means. Could you show me what the updated line of code would look like?

Reply:
Apology for the confusion. I saw you've renamed the `random_pet.eventhubnamespace_name` to `random_pet.eventhub_namespace_name` in line 38 and it looks good to me now. Thanks.